FeedAPIChallenge/Feed API/RemoteFeedLoaderMapper.swift

```diff
@@ -0,0 +1,40 @@
+
+import Foundation
+
+internal final class RemoteFeedImagesMapper {
+	private struct Root: Decodable {
+		let remoteFeedImages: [RemoteFeedImage]
+
+		enum CodingKeys: String, CodingKey {
+			case remoteFeedImages = "items"
+		}
```

Comment:
CodingKeys is not needed in this context if you name the property `items`.

This way, the code is shorter and more expressive:

```suggestion
		let items: [RemoteFeedImage]
```